p4src/linux_networking/README_LINUX_NETWORKING_WITH_ECMP.md

```diff
@@ -133,34 +134,56 @@ Note:
     ip link set dev TEP1 up
 ```
 
-#### 9. Set the pipeline and add br-int to OVS
+#### 9. Set the pipeline
 ```
     p4rt-ctl set-pipe br0 lnw.pb.bin p4Info.txt
-    ovs-vsctl add-br br-int
-    ip link set dev br-int up
 ```
 
-#### 10. Configure VXLAN port
+#### 10. Start and run ovs-vswitchd server and ovsdb-server
+Kill any existing ovs process if running.
+```
+mkdir -p $IPDK_RECIPE/install/var/run/openvswitch
+rm -rf $IPDK_RECIPE/install/etc/openvswitch/conf.db
+
+sudo $IPDK_RECIPE/install/bin/ovsdb-tool create $IPDK_RECIPE/install/etc/openvswitch/conf.db $IPDK_RECIPE/install/share/openvswitch/vswitch.ovsschema
+
+export RUN_OVS=$IPDK_RECIPE/install
+
+sudo $IPDK_RECIPE/install/sbin/ovsdb-server \
+	        --remote=punix:$RUN_OVS/var/run/openvswitch/db.sock \
+	        --remote=db:Open_vSwitch,Open_vSwitch,manager_options \
+                --pidfile --detach
+sudo $IPDK_RECIPE/install/sbin/ovs-vswitchd --detach --no-chdir unix:$RUN_OVS/var/run/openvswitch/db.sock --mlockall --log-file=/tmp/ovs-vswitchd.log
```

Comment:
The sample code would be easier to read if you separated long lines into shorter continued lines (as is done with ovsdb-server above). For example,

```bash
sudo $IPDK_RECIPE/install/sbin/ovs-vswitchd --detach --no-chdir \
    unix:$RUN_OVS/var/run/openvswitch/db.sock --mlockall \
    --log-file=/tmp/ovs-vswitchd.log
```